Software/a314fs/messages.h

```diff
@@ -315,4 +315,20 @@ struct UnsupportedResponse
 	short error_code;
 };
 
+struct DiskInfoRequest
```

Comment:
Perhaps change the name of the struct to DiskSizeInfoRequest, since the message is called ACTION_DISK_SIZE_INFO ?